From a31cd27475f0b609b224a38564c993d7f01948d1 Mon Sep 17 00:00:00 2001
From: Jiri Benc <jbenc@redhat.com>
Date: Fri, 12 Sep 2014 14:29:47 -0400
Subject: [net] openvswitch: flow_netlink: Use pr_fmt to OVS_NLERR output

Message-id: <b1ff30c0df6aebbdcd3549e1a47d38473e8277be.1410531783.git.jbenc@redhat.com>
Patchwork-id: 94028
O-Subject: [RHEL7.1 net PATCH 14/50] openvswitch: flow_netlink: Use pr_fmt to OVS_NLERR output
Bugzilla: 1110384
RH-Acked-by: Flavio Leitner <fbl@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110384
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7948291

commit 2235ad1c3ac545bd8fc2c026be5be16d98b9a891
Author: Joe Perches <joe@perches.com>
Date:   Mon Feb 3 17:18:21 2014 -0800

    openvswitch: flow_netlink: Use pr_fmt to OVS_NLERR output

    Add "openvswitch: " prefix to OVS_NLERR output
    to match the other OVS_NLERR output of datapath.c

    Signed-off-by: Joe Perches <joe@perches.com>
    Signed-off-by: Jesse Gross <jesse@nicira.com>

Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/net/openvswitch/flow_netlink.c b/net/openvswitch/flow_netlink.c
index 1b22ad2..7bd09b7 100644
--- a/net/openvswitch/flow_netlink.c
+++ b/net/openvswitch/flow_netlink.c
@@ -16,6 +16,8 @@
  * 02110-1301, USA
  */
 
+#define pr_fmt(fmt) KBUILD_MODNAME ": " fmt
+
 #include "flow.h"
 #include "datapath.h"
 #include <linux/uaccess.h>
-- 
1.7.1